NuRadioMC/simulation/simulation.py

```diff
@@ -1622,6 +1632,12 @@ def run(self):
                         channel[chp.Vrms_NuRadioMC_simulation] = self._Vrms_per_channel[station_id][channel.get_id()]
                         channel[chp.bandwidth_NuRadioMC_simulation] = self._integrated_channel_response[station_id][channel.get_id()]
 
+                        # Always store this, even when no "extra" trigger channels are simulated
+                        if self.__trigger_channel_ids is not None and channel.get_id() in self.__trigger_channel_ids and channel.get_id() in self._Vrms_per_trigger_channel[station_id]:
+                            channel[chp.Vrms_trigger_NuRadioMC_simulation] = self._Vrms_per_trigger_channel[station_id][channel.get_id()]
+                        else:
+                            channel[chp.Vrms_trigger_NuRadioMC_simulation] = 0
```

Comment:
I'm slightly confused here... if there are no 'additional trigger channels' RNO-G style, this parameter is still set, but it's set to 0? I would have thought it makes more sense to either default it to the Vrms of the 'normal' channel if there is no additional trigger channel associated, or to just not set it. But probably I'm missing something?

Reply:
I did not want to set it to the same value as the normal channel and I think it made it easier to just set it to zero in case there are no explicit trigger channels.

Reply:
Fine?

Reply:
Unless I'm still missing something I would suggest not setting this. Having it set to an 'invalid' value like 0 will either lead to problems anyway (e.g. division by 0 for SNR calculations) or lead to wrong conclusions (e.g. the user thinks the trigger channels were simulated without noise if there are no additional trigger channels).

Reply:
I am pretty sure I did it for a (good) reason. But lets see. I can take it of an run it again.

Reply:
Okay, I am now only storing the attribute for the trigger channels.